tests/ImageSharp.Tests/Formats/Jpg/JpegTests.cs

```diff
@@ -17,62 +17,45 @@ public class JpegTests
 
         public JpegTests(ITestOutputHelper output)
         {
-            Output = output;
+            this.Output = output;
         }
+        public static IEnumerable<string> AllJpegFiles => TestImages.Jpeg.All;
 
-        protected string CreateTestOutputFile(string fileName)
-        {
-            if (!Directory.Exists(TestOutputDirectory))
-            {
-                Directory.CreateDirectory(TestOutputDirectory);
-            }
-
-            //string id = Guid.NewGuid().ToString().Substring(0, 4);
-
-            string ext = Path.GetExtension(fileName);
-            fileName = Path.GetFileNameWithoutExtension(fileName);
-
-            return $"{TestOutputDirectory}/{fileName}{ext}";
-        }
-
-        protected Stream CreateOutputStream(string fileName)
-        {
-            fileName = CreateTestOutputFile(fileName);
-            Output?.WriteLine("Opened for write: "+fileName);
-            return File.OpenWrite(fileName);
-        }
-
-        public static IEnumerable<object[]> AllJpegFiles
-            => TestImages.Jpeg.All.Select(file => new object[] {TestFile.Create(file)});
-
+        // TODO: Turned off PixelTypes.All to be CI-friendly, what should be the practice?
```

Comment:
See the output here when I'm doing crazy things

https://github.com/JimBobSquarePants/ImageSharp/issues/27